pubobot/bot.py

```diff
@@ -918,7 +918,7 @@ async def processmsg(self, msg):
             elif lower[0] == "subfor":
                 await self.subfor(member, lower[1:2])
 
-            elif lower[0] == "teams":
+            elif lower[0] == ["teams", "turn"]:
```

Comment:
Need to use `in`:

```python
lower[0] in ["teams", "turn"]
```

Reply:
oh good point, shouldn't commit real fast 3 am I guess 🤤